packages/babel-parser/ast/spec.md

```diff
@@ -36,7 +37,7 @@ These are the core @babel/parser (babylon) AST node types.
     - [DoWhileStatement](#dowhilestatement)
     - [ForStatement](#forstatement)
     - [ForInStatement](#forinstatement)
-    - [ForOfStatement](#forofstatement)
+  - [ForOfStatement](#forofstatement)
```

Comment:
Are these indent changes auto-generated? :thinking:

Reply:
Yeah, I didn't change anything about the list, I just added my change in spec and the list was generated for me 🤔

Reply:
I don't think this file is autogenerated, maybe it was one of your ide extensions?
Can you revert this line and the on above in `Changes`?

Reply:
You are right. Apparently, `Markdown All in One` was doing auto formatting.